packages/react/src/context.tsx

```diff
@@ -11,4 +11,9 @@ export function useParent(): ParentContext {
   return parent
 }
 
+export function useParentOptional(): ParentContext {
```

Comment:
We could rename the other hook to `useParentOrThrow` and this one to `useParent`. Happy with whatever.